apps/constellation/constellation.cpp

```diff
@@ -272,20 +271,17 @@ Constellation::Constellation(CertificatePtr certificate, Config config)
   FETCH_LOG_INFO(LOGGING_NAME, "");
 
   // Enable experimental features
-  if (cfg_.features.IsEnabled("synergetic"))
-  {
-    assert(dag_);
-    dag_service_ = std::make_shared<ledger::DAGService>(muddle_.AsEndpoint(), dag_);
-    reactor_.Attach(dag_service_->GetWeakRunnable());
+  assert(dag_);
+  dag_service_ = std::make_shared<ledger::DAGService>(muddle_.AsEndpoint(), dag_);
+  reactor_.Attach(dag_service_->GetWeakRunnable());
 
-    auto syn_miner = std::make_unique<NaiveSynergeticMiner>(dag_, *storage_, certificate);
-    if (!reactor_.Attach(syn_miner->GetWeakRunnable()))
-    {
-      FETCH_LOG_ERROR(LOGGING_NAME, "Failed to attach synergetic miner to reactor.");
-      throw std::runtime_error("Failed to attach synergetic miner to reactor.");
-    }
-    synergetic_miner_ = std::move(syn_miner);
+  auto syn_miner = std::make_unique<NaiveSynergeticMiner>(dag_, *storage_, certificate);
```

Comment:
For the moment I would like to keep the synergetic miner controlled via feature flag.

Reply:
Done.